### PR TITLE
Created "oracle vm box" topic

### PR DESCRIPTION
**Pull request by Sigmale1000**
### Please confirm this pull request meets the following requirements:

- [X] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [X] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [X] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Curating a new topic or collection

- [X] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [X] My folder contains a `*.png` image (if applicable) and `index.md`
- [X] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> VM Box is getting used a lot for developing

### CODE REVIEW

File: topics/slack/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (smashwilson) [inline topics/slack/index.md:1]: Looks like the Slack files slipped into this PR as well ☝🏻

Comment (Sigmale1000) [reply topics/slack/index.md:1]: Will change it 👍🏻

**Review by smashwilson — APPROVED** [review]
😎 Perfect. Thanks!

**Comment by smashwilson** [comment]
Hah, approved too soon.

That was weird though - it looked like both files had been deleted from the diff... ?

**Comment by smashwilson** [comment]
Oh, I bet you need to merge in `main` and resolve the conflicts.

**Comment by Sigmale1000** [comment]
Give me a sec. I'm kinda lost.

**Comment by Sigmale1000** [comment]
All fixed now 👍🏻

**Review by smashwilson — APPROVED** [review]
♻️ 👍🏻

**Pull request merged**